<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ini</class>
 <widget class="QWidget" name="Ini">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ini</string>
  </property>
  <widget class="QPushButton" name="pushButtonChooseFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать файл</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>380</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>Демонстрация данных</string>
   </property>
   <widget class="QPushButton" name="pushButtonShowAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать содержимое всего
 ini-файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowRangeSections">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать диапазон секций</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowOneSection">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать одну секцию</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>380</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>Работа с данными</string>
   </property>
   <widget class="QPushButton" name="pushButtonChangeContentSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить данные секции</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddNewSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить новую секцию </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteData">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить данные</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>780</width>
     <height>730</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreateFile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>380</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
